HttpClients/SpoonacularClient.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Microsoft.AspNetCore.WebUtilities;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace JifBot.HttpClients
+{
+    public interface ISpoonacularClient
+    {
+        public Task<SearchResponse> GetSauceRecipe(string sauceSearchQuery);
+    }
+
+    public class SpoonacularClient : ISpoonacularClient
+    {
+        private readonly HttpClient _httpClient;
+
+        public SpoonacularClient(HttpClient httpClient)
+        {
+            _httpClient = httpClient;
+            _httpClient.BaseAddress = new Uri("https://api.spoonacular.com/");
+        }
+
+        public async Task<SearchResponse> GetSauceRecipe(string sauceSearchQuery = "tomato")
+        {
+            var queryParameters = new Dictionary<string, string>()
+            {
+                { "apiKey", "db3b318ab2c04661b9fa2f541cbcef43" },
```

Comment:
I usually keep api keys in the database. An example:
https://github.com/Vertigeux/JifBot/blob/master/Commands/Information.cs#L258